lib/ProductOpener/Products.pm

```diff
@@ -1001,6 +1001,42 @@ sub compute_sort_keys ($product_ref) {
 	return;
 }
 
+=head2 update_existing_product_without_creating_a_new_revision( $mongodb_to_mongodb, $product_ref, $products_collection, $data_root, $path)
+
+update existing product without creating a new revision
+
+=head3 Parameters
+
+=head4 $mongodb_to_mongodb
+  
+=head4 $product_ref
+
+Product object reference.
+
+=head4  $products_collection
+=head4 $data_root
```

Comment:
$data_root is a global variable, you can use it without passing it as an argument to the function